### ISSUE TITLE
WARNING: Inst already refers to: cljs.core/Inst

### ISSUE THREAD

**Issue by tmarble** [plumatic/schema#367]
With `[prismatic/schema "1.1.3"]` and `[org.clojure/clojurescript "1.9.93" :scope "provided"]`
loading schema.core provokes this warning:

````
WARNING: Inst already refers to: cljs.core/Inst being replaced by: schema.core/Inst at line 398 out/schema/core.cljs
````

The CLJS definition is [here](https://github.com/clojure/clojurescript/blob/master/src/main/cljs/cljs/core.cljs#L1272)

The Schema definition is [here](https://github.com/plumatic/schema/blob/master/src/cljx/schema/core.cljx#L398)

And we still get the warning despite the [exclude](https://github.com/plumatic/schema/blob/master/src/cljx/schema/core.cljx#L78) ???

**Comment by w01fe** [comment]
Hmmm, haven't seen that before.  Are you sure you aren't getting an earlier
version of Schema through a transitive dependency?

**Comment by tmarble** [comment]
Gah! That was it! I hadn't pushed one of my newly updated libraries (using Schema) to my local repo prior to testing the application.

Thanks!

**Comment by w01fe** [comment]
Great, thanks for the update!
